core/tests/player_test.py

```diff
@@ -60,5 +61,30 @@ def test_add_species__removes_discard_card_from_player_hand_cards(self):
             self.player.hand_cards
         )
 
-    # Todo: write add_species position left and right tests.
+    def test_add_species__adds_species_to_the_left_of_existing_species(self):
+        self.player.hand_cards = [TraitCard(name='Trait 1')]
+        initial_species = [Species(name='first species')]
+        self.player.species = initial_species
+        self.player.add_species(
+            discard_card=self.player.hand_cards[0],
+            position=SpeciesPosition.LEFT
+        )
 
+        self.assertEqual(
+            ['nameless species', 'first species'],
+            [str(self.player.species[0].name), str(self.player.species[1].name)]
```

Comment:
Do these actually need to be cast to `str`?

Reply:
No. Updated.